docs/Extension.md

```diff
@@ -0,0 +1,96 @@
+# Extension
```

Comment:
I think `Extensibility.md` file name is more generic.

Reply:
Renamed.